R/circacompare.R

```diff
@@ -11,6 +11,7 @@
 #' @param alpha_threshold The level of alpha for which the presence of rhythmicity is considered. Default is 0.05.
 #' @param timeout_n The upper limit for the model fitting attempts. Default is 10,000.
 #' @param control \code{list}. Used to control the parameterization of the model.
+#' @param sample_weights A numeric vector of per-sample weights to allow downweighting of outlier samples.
```

Comment:
please make the same change as to `circa_single()` docs please and argument name to become `weights` etc.

Reply:
it's done